components/Author.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+import AuthorCollection from '../interfaces/AuthorCollection'
+import { MarkdownFileWithUrl } from '../pages/authors'
+
+import ContentCardList from './ContentCardList'
+import PageLayout from './PageLayout'
+
+export interface Props {
+ author: AuthorCollection
+ records: MarkdownFileWithUrl[]
+}
+
+const Author: React.FunctionComponent<Props> = props => {
+  const authorName = props.validAuthor.name
+  return (
+      <PageLayout documentTitle={`Resources by ${authorName}`} appendSiteTitle={true}>
```

Comment:
I don't think we need the language of `Resources by` — I don't think the word "resources" is clear language for an end user, and I think if we present a page with the author's name and links to items they authored it will be visually and organizationally clear.

Reply:
I had suggested "resources" here because I thought it would be a better title than just `authorName` but I agree with your thinking

Reply:
Yeah, I saw, it went away before I could comment!

Reply:
Thanks Lisa. I will update the language and I will also add a photo of what this looks like.

Reply:
The PR has been updated with screenshots, please have one more look, thanks.
cc: @marekweb